src/app/components/core/control-dispatcher/control-dispatcher.module.ts

```diff
@@ -94,7 +94,7 @@ const routes: Routes = [
     canActivate: [RolesGuard]
   },
   {
-    path: 'entry-point/:name',
+    path: 'entry-point/:name/details',
```

Comment:
I believe `\details` should also be added in the path of `ControlScheduleTaskDetailsComponent` in this module